perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/GroupsManagerBlImpl.java

```diff
@@ -1716,8 +1716,8 @@ protected boolean hasCandidateExistingMember(Candidate candidate, RichMember ric
 			membersSource = getGroupMembersExtSourceForSynchronization(sess, group, source);
 
 			//Prepare info about userAttributes which need to be overwritten (not just updated) and memberAttributes which need to be merged not overwritten
-			List<String> overwriteUserAttributesList = getOverwriteUserAttributesListFromExtSource(membersSource);
-			List<String> mergeMemberAttributesList = getMemberAttributesListToBeMergedFromExtSource(membersSource);
+			List<String> overwriteUserAttributesList = getAttributesListFromExtSource(membersSource, "overwriteUserAttributes");
+			List<String> mergeMemberAttributesList = getAttributesListFromExtSource(membersSource, "mergeMemberAttributes");
```

Comment:
I would suggest creating some constants for these string values.

Reply:
Added.